ast/term.go

```diff
@@ -556,6 +556,15 @@ func IsScalar(v Value) bool {
 	return false
 }
 
+// IsIterable returns true if the AST value is an iterable type.
+func IsIterable(x Value) bool {
```

Comment:
Do we really need to expose this? 🤔 I'd think it work make for a decent helper method in topdown/eval.go, too.

Reply:
Sure, I can move it there 👍